README.md

```diff
@@ -1,605 +1,19 @@
-#include "app_links_windows/app_links_windows_plugin.h"
-```
-
-Add this method in <PROJECT_DIR>\windows\runner\win32_window.cpp
-
-```cpp
-bool Win32Window::SendAppLinkToInstance(const std::wstring& title) {
-  // Find our exact window
-  HWND hwnd = ::FindWindow(kWindowClassName, title.c_str());
-
-  if (hwnd) {
-    // Dispatch new link to current window
-    SendAppLink(hwnd);
-
-    // (Optional) Restore our window to front in same state
-    WINDOWPLACEMENT place = { sizeof(WINDOWPLACEMENT) };
-    GetWindowPlacement(hwnd, &place);
-    switch(place.showCmd) {
-      case SW_SHOWMAXIMIZED:
-          ShowWindow(hwnd, SW_SHOWMAXIMIZED);
-          break;
-      case SW_SHOWMINIMIZED:
-          ShowWindow(hwnd, SW_RESTORE);
-          break;
-      default:
-          ShowWindow(hwnd, SW_NORMAL);
-          break;
-    }
-    SetWindowPos(0, HWND_TOP, 0, 0, 0, 0, SWP_SHOWWINDOW | SWP_NOSIZE | SWP_NOMOVE);
-    SetForegroundWindow(hwnd);
-    // END Restore
-
-    // Window has been found, don't create another one.
-    return true;
-  }
-
-  return false;
-}
-```
-
-Add the call to the previous method in `CreateAndShow`
-
-```cpp
-bool Win32Window::CreateAndShow(const std::wstring& title,
-                                const Point& origin,
-                                const Size& size) {
-if (SendAppLinkToInstance(title)) {
-    return false;
-}
-
-...
-```
-
-At this point, you can register your own scheme.  
- On Windows, URL protocols are setup in the Windows registry.
-
-This package won't do it for you.
-
-You can achieve it with [url_protocol](https://pub.dev/packages/url_protocol) inside you app.
-
-The most relevant solution is to include those registry modifications into your installer to allow
-for deregistration.
-
-</details>
-
-#### For Mac OS
+# `Supabase Flutter`
 
-<details>
-  <summary>How to setup</summary>
+Flutter Client library for [Supabase](https://supabase.com/).
 
-Add this XML chapter in your macos/Runner/Info.plist inside <plist version="1.0"><dict> chapter:
+- Documentation: https://supabase.com/docs/reference/dart/introduction
 
-```xml
-<!-- ... other tags -->
-<plist version="1.0">
-<dict>
-  <!-- ... other tags -->
-  <key>CFBundleURLTypes</key>
-  <array>
-      <dict>
-          <key>CFBundleURLName</key>
-          <!-- abstract name for this URL type (you can leave it blank) -->
-          <string>sample_name</string>
-          <key>CFBundleURLSchemes</key>
-          <array>
-              <!-- your schemes -->
-              <string>sample</string>
-          </array>
-      </dict>
-  </array>
-  <!-- ... other tags -->
-</dict>
-</plist>
-```
+## Run locally
 
-</details>
+This repo is a monorepo powered by [Melos](https://melos.invertase.dev/) containing [supabase_flutter](https://github.com/supabase/supabase-flutter/tree/main/packages/supabase_flutter) and its sub-libraries. All packages are located in the `packages` directory.
 
----
+To install on a locally developed app:
+- Clone this repo
```

Comment:
I think we can add the simple command to install melos here as well:
`dart pub global activate melos`